### PR TITLE
Bug fixes for ME21-B2 examples.

### PR DESCRIPTION
**Pull request by Jacob-Scheiffler**
Most of the ME21-B2 examples worked out of the box, however a few examples required these small fixes.

### CODE REVIEW

**Review by sihyung-maxim — APPROVED** [review]

File: Libraries/CMSIS/Device/Maxim/MAX32672/Include/max32672.h [hunk 1]
```diff
@@ -307,7 +307,7 @@ mxc_aes_key_regs_t mxc_sys_aes_key_regs_t;
 
 /******************************************************************************/
 /*                                                                        AES */
-#define MXC_BASE_AES ((uint32_t)0x40007400UL)
+#define MXC_BASE_AES ((uint32_t)0x40207400UL)
```
Comment (lorne-maxim) [inline Libraries/CMSIS/Device/Maxim/MAX32672/Include/max32672.h:310]: This is not something that changed for B2 is it?  All revisions should be using this address?

File: Libraries/PeriphDrivers/Source/SYS/pins_me21.c [hunk 2]
```diff
@@ -68,8 +68,8 @@ const mxc_gpio_cfg_t gpio_cfg_uart0_flow = { MXC_GPIO0, (MXC_GPIO_PIN_10 | MXC_G
                                              MXC_GPIO_FUNC_ALT1, MXC_GPIO_PAD_NONE };
 const mxc_gpio_cfg_t gpio_cfg_uart0_flow_disable = { MXC_GPIO0, (MXC_GPIO_PIN_10 | MXC_GPIO_PIN_11),
                                                      MXC_GPIO_FUNC_IN, MXC_GPIO_PAD_WEAK_PULL_UP };
-const mxc_gpio_cfg_t gpio_cfg_uart1 = { MXC_GPIO0, (MXC_GPIO_PIN_2 | MXC_GPIO_PIN_3),
-                                        MXC_GPIO_FUNC_ALT2, MXC_GPIO_PAD_NONE };
+const mxc_gpio_cfg_t gpio_cfg_uart1 = { MXC_GPIO0, (MXC_GPIO_PIN_28 | MXC_GPIO_PIN_29),
```
Comment (lorne-maxim) [inline Libraries/PeriphDrivers/Source/SYS/pins_me21.c:71]: Same question.  Should these changes have been there all along, or are these specific to the B2 revision?

File: Libraries/CMSIS/Device/Maxim/MAX32672/Include/max32672.h (hunk 1 above)
Comment (lorne-maxim) [reply Libraries/CMSIS/Device/Maxim/MAX32672/Include/max32672.h:310]: Jacob confirmed this change applies to all revs.

File: Libraries/PeriphDrivers/Source/SYS/pins_me21.c (hunk 2 above)
Comment (lorne-maxim) [reply Libraries/PeriphDrivers/Source/SYS/pins_me21.c:71]: Jacob confirmed this change applies to all revs.

**Pull request merged**